adapters/nativo/nativo_test.go

```diff
@@ -0,0 +1,21 @@
+package nativo
+
+import (
+	"testing"
+
+	"github.com/prebid/prebid-server/v2/adapters/adapterstest"
+	"github.com/prebid/prebid-server/v2/config"
+	"github.com/prebid/prebid-server/v2/openrtb_ext"
+)
+
+func TestBidderNativo(t *testing.T) {
+	bidder, buildErr := Builder(openrtb_ext.BidderNativo, config.Adapter{
+			Endpoint: "https://foo.io/?src=prebid"},
```

Comment:
why is it "https://foo.io/?src=prebid" ? 
It should be endpoint from yaml

Reply:
done